Add Linters

AztkMarty opened the pull request:
This feature was requested by Microverse reviewers in the community. The code changes have been tested locally and all tests have passed successfully.
Please review and let me know if there are any changes or improvements that need to be made. Thanks for your time.

AztkMarty committed:
Add Linters

Strangeal reviewed (changes requested):
Hi @AztkMarty, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![Alt Text](https://media1.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif)

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
**Congratulations 🎉 🥳 on the impressive work you've done so far. I've made some suggestions below to help you improve your project so kindly take a look at it and if you don't understand anything please don't hesitate to ask. Best of luck!🤞 🍀. Using `inline` comments would have been the best but since I don't have access to your `index.html` and `style.css` in this pull request. I will do my best to explain the changes here. Let's get started. 😊** 

- [x] Kindly adjust the spacing above the `hamburger` menu to match that of the `Figma`design template so that your design will look nicer and create a good user experience.
<table>
<tr>
<th align="middle">Fogma design</th>
<th align="middle">Your design</th>
</tr>
<td>
<img width="381" alt="Screenshot 2023-04-28 at 04 44 59" src="https://user-images.githubusercontent.com/107192076/235058237-5c964346-2678-4914-918c-e748aa8896f0.png">
</td>
<td>
<img width="373" alt="Screenshot 2023-04-28 at 04 45 26" src="https://user-images.githubusercontent.com/107192076/235058276-43d00107-8a6b-4005-90a1-03d88d7c7247.png">
</td>
</table>

- [x]  Kindly make sure to use an `h1` tag to display the main heading of your portfolio. https://github.com/AztkMarty/Portfolio-Setup/blob/62c9e24137a6ee1f4304c8935103fee484ce5014/index.html#L25
- It is important to have only one `h1` tag on a web page because the `h1` tag serves as the main heading of the page and gives an overall context of what the page is about. Search engines also use the `h1` tag to understand the topic of the page and determine its relevance to a search query. If there are multiple h1 tags on a page, it can confuse search engines and lead to lower visibility in search results. In addition, having multiple `h1` tags can also make it difficult for users with screen readers or other assistive technologies to understand the structure and content of the page.

- [x] Kindly make sure to adhere to the `Figma` design template as much as possible. In other, for you to do that please remove the `list-styles` you have on your `li` tags.
https://github.com/AztkMarty/Portfolio-Setup/blob/62c9e24137a6ee1f4304c8935103fee484ce5014/index.html#L33-L59
<table>
<tr>
<th align="middle">Fogma design</th>
<th align="middle">Your design</th>
</tr>
<td>
<img width="378" alt="Screenshot 2023-04-28 at 04 49 47" src="https://user-images.githubusercontent.com/107192076/235061269-c01fc684-6e6e-4919-8de9-98343bbbabce.png">
</td>
<td>
<img width="373" alt="Screenshot 2023-04-28 at 04 49 14" src="https://user-images.githubusercontent.com/107192076/235061338-2b746bf2-b3dc-4733-8d7a-6ef4448954b7.png">
</td>
</table>

- [x] Also, make sure to include `alt` text in your images. Including `alt` text in their images is not only good for `accessibility,` but it's also an essential part of good web development practice. The `alt` attribute is used to describe the contents of an image to users who may not be able to see the image. This includes users with visual impairments, users who use screen readers, and users who may have slow internet connections or have turned off image loading to save bandwidth.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AztkMarty committed:
Improve accessibility and adhere to design template

AztkMarty commented:
Hi there! I've made some changes to improve the accessibility of the site and ensure that it adheres to the Figma design template. Specifically, I added an h1 tag for the main heading, removed list-styles from li tags, and included alt text in images. Let me know if you have any questions or concerns. Thank you!

binhussen commented on index.html:
```diff
@@ -12,7 +12,7 @@
             <nav>
-                <a href="!" class="menu-box">
+                <a href="!" class="menu-box flex">
                     <img class="menu-icon" src="images/menu.png">
```
- Kindly, adjust the space above `hamburger` as the previous code reviewer suggested, I think you are using the wrong image for your `nav`, use only the hamburger icon instead of using an image for your `nav` which will help you later 👍

binhussen commented on index.html:
```diff
@@ -33,27 +33,27 @@ <h2 id="title2">I'm a software developer</h2>
-                        <img src="images/twitter.svg">
+                        <img src="images/twitter.svg" alt="Twitter icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/Linkedin icon.svg">
+                        <img src="images/Linkedin icon.svg" alt="Linkedin icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/medium.svg">
+                        <img src="images/medium.svg" alt="Medium icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/github.svg">
+                        <img src="images/github.svg" alt="GitHub icon to be in contact">
                     </a>
                 </li>
                 <li>
                     <a href="!">
-                        <img src="images/angellist.svg">
+                        <img src="images/angellist.svg" alt="Angellist icon to be in contact">
                     </a>
                 </li>
```
-[Optional] Great job for improving the previous reviewer suggestions  👍 , I think it would be better if you minimize the gap between social icons and the the little arrow to stick to the given design and have a better UI 👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/235227328-b4c52096-a7d8-4d7f-aa5f-84ae7ae8650c.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/235227434-bde67ef2-32af-4132-855a-b8dc44689018.png" width="200">|


😎️

🤝️

binhussen reviewed (changes requested):
Hi @AztkMarty , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] Kinndly, remove the broken logo from your readme or replace it with your own to make the readme more professional.
- [Optional] Please use a descriptive branch name that describes what you have done to be professional 👍

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AztkMarty committed:
Added logo to README and updated mobile menu logo to match Figma template.

AztkMarty commented:
In this Milestone, I introduced the following changes
-Added logo to README and updated mobile menu logo to match Figma template.
-Updated branch name to better reflect changes made
-README logo added
-This comment is to inform that a new PR was created to reflect the changes made to the branch name. This pull request was closed when the branch name was changed, but it has now been reopened.

stanleySimeon reviewed (changes requested):
# Hi @AztkMarty 
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media2.giphy.com/media/vQGBDTkhtIunaJYmRF/giphy-downsized.gif?cid=29caca75sf3uq9y4qgulk5ybbojwg94b4zo6temfzeel417x&ep=v1_gifs_search&rid=giphy-downsized.gif&ct=g
)

## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

# Required Changes ♻️

* [x] As mentioned by the previous reviewer, Kindly, adjust the space above the hamburger, I think you are using the wrong image for your nav, use only the hamburger icon instead of using an image for your nav which will help you later 👍

* [OPTIONAL] As mentioned by the previous reviewer, I highly recommend you minimize the gap between social icons and the little arrow to stick to the given design and have a better UI. 👌 Kindly follow the Figma template using this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=tzeM6YqVWwlYUvkB-0) to update your project design.

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AztkMarty committed:
Replace image with original icon and fix social media gaps

AztkMarty commented:
##In this Milestone, I introduced the following changes
-Replaced the image used for the navigation menu with the original icon of the page and fixed the spacing above, which improved the visual consistency of the design.
-Corrected the gaps between the social media icons by adjusting the spacing with CSS, making them appear more evenly spaced and centered on the Y axis.

Efiamotu-1 commented on README.md:
[OPTIONAL]
> - I just want to note a few things which i would like you to take into your next milestone as it will help you be a better developer
- [x] Firstly after a change is requested you do not need to rewrite your pr summary using the comments like you did in this pr thread, all you need to do is to use just the one initial pr summary and make sure to be descriptive with it detailing any new or previous changes added. :grin: 
- [x] next going further you are also required to always name your branch descriptively in relation to the milestone you are working or the changes your are going to make in that branch for example names like `add_works_section` is a good branch name when you are working on creating a works section for your portfolio using names like `linters` is not professional so please take note. your earlier reviewers were graceful enough to still review your projects for this error, in your following projects this might not be so, so just take note of this :+1: 
- [x] lastly you have done well implementing the requirements of the previous reviewers and i am glad to approve your pr so you can complete your projects for the week, Goodluck :+1: :100:

Efiamotu-1 reviewed (approved):
Hi @AztkMarty , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

![giphy](https://user-images.githubusercontent.com/89055502/234114221-4ca295de-fd8f-4af6-9c49-a6d4c6ea67f7.gif)


## Highlights :dart: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
